omod/src/main/resources/liquibase.xml

```diff
@@ -195,5 +195,31 @@
 							 tableName="metadatamapping_metadata_source"
 							 columnNames="name" />
 	</changeSet>
-
+	
+	<changeSet id="metadatamapping-2016-01-06-0800" author="jasonvena">
+		<preConditions onFail="MARK_RAN">
+			<columnExists tableName="metadatamapping_metadata_term_mapping"
+						  columnName="metadata_class" />
+		</preConditions>
```

Comment:
Hi Mikko, I have intended it as written, following examples from core developers such as seen [here](https://github.com/openmrs/openmrs-core/blob/cdbf9421212f5f5d48fa9d92c42515e4c517f20c/api/src/main/resources/liquibase-update-to-latest.xml#L7696) and [here](https://github.com/openmrs/openmrs-core/blob/cdbf9421212f5f5d48fa9d92c42515e4c517f20c/api/src/main/resources/liquibase-update-to-latest.xml#L3149).